persistence/genesis.go

```diff
@@ -46,8 +46,7 @@ func (m *persistenceModule) populateGenesisState(state *genesis.GenesisState) {
 		}
 	}
 	for _, pool := range state.GetPools() {
-		poolNameBytes := []byte(pool.GetAddress())
-		err = rwContext.InsertPool(pool.GetAddress(), poolNameBytes, pool.GetAmount())
+		err = rwContext.InsertPool(pool.GetAddress(), pool.GetAmount())
```

Comment:
Please add a comment that the `address` for pools is actually the name.

You can find more details on it in [here](https://github.com/pokt-network/pocket-network-protocol/blob/issues/23/aat/utility/README.md#373-pool) or see the `genesis.json` file for reference:

```json
{
  "accounts": [
    {
      "address": "6f66574e1f50f0ef72dff748c3f11b9e0e89d32a",
      "amount": "100000000000000"
    },
  ],
  "pools": [
    {
      "address": "DAO",
      "amount": "100000000000000"
    },
  ],
```